### ISSUE TITLE
Where is the .pkg file?

### ISSUE THREAD

**Issue by MacProdromos** [synapsepd/MacOS-OneDrive-KFM#2]
Hi - I am a newbie and I am struggling to go through the guide for MacOS-OneDrive-KFM article for the reasons that first, I cannot find the OutSet installation file on step 3. Then secondly on step 4, I cannot find where I need to get "MacOS-OneDrive-KFM-X. X.pkg" from? 

Can you please assist somehow making it a bit clearer?

Thank you!
Prodromos

**Comment by notanewbie** [comment]
Devs here like to post only source code in the repo, compiling the application/installation files seperately and posting them in the repo as releases. The same is the case with these two applications.

Go to the main tab for these repos (where it says "<>Code") and click the part on the page that says "1 Release" or "X Releases". There should be at least one release. There, you can pick a version, click Assets for the version you want, and download the file.

To make this easier for you I'm going to copy the links and paste them here:

MacOS-OneDrive-KFM 0.1: https://github.com/synapsepd/MacOS-OneDrive-KFM/releases/download/0.1/MacOS-OneDrive-KFM-0.1.pkg

Outset 2.0.6: https://github.com/chilcote/outset/releases/download/v2.0.6/outset-2.0.6.pkg

**Comment by maysarh** [comment]
I'm not a professional here, but I followed the steps except the last step for building the package, I really don't know how to make it.
I'm coping the git https and trying to paste in terminal with git clone but it showing this message:

**fatal: destination path 'MacOS-OneDrive-KFM' already exists and is not an empty directory.**

even I can't understand the note for Munki-pkg "ensure it is in your path" I only got the **munkitools-5.5.0.4362.pkg** and directly install it.

appreciate your support here. 🙏🏻
